src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -306,6 +307,18 @@ class InferenceProfiler {
     return cb::Error::Success;
   }
 
+  cb::Error ProfilePeriodicConcurrencyMode()
+  {
+    auto& manager{dynamic_cast<PeriodicConcurrencyManager&>(*manager_)};
+    std::vector<RequestRecord> request_records{manager.RunExperiment()};
+
+    InferenceLoadMode id{1, 0.0};
```

Comment:
(1, 0.0) is this a magic number?

Reply:
This line is only necessary because the current collector class was designed to require an ID.

This ID is meaningless, however. I'm just trying to supply a required argument to a method so that I can make use of the other logic in the method.

There is future work needed to redesign the collector class to work better with this new mode.

Reply:
Added FIXME comment 👍